node/scripts/copy_repo.sh

```diff
@@ -9,7 +9,7 @@ set -euo pipefail
 
 SCRIPT_DIR=$(dirname "$0")
 cd "${SCRIPT_DIR}"/..
-. bin/build_helpers.sh
+. ../bin/build_helpers.sh
```

Comment:
Nitpick: the intent of this prefix for the other scripts was that they always run from the root directory, i.e. change the `cd` instead. You don't have to do that but I think it could make things simpler.